test/gurka/test_graphfilter.cc

```diff
@@ -579,3 +579,154 @@ TEST(Standalone, FilterTestMultipleEdges) {
+  auto result = gurka::do_action(valhalla::Options::route, map, {"A", "F"}, "bicycle");
+  gurka::assert::raw::expect_path(result, {"AC", "CF"});
+
+  {
+    GraphReader graph_reader = GraphReader(map.config.get_child("mjolnir"));
+
+    GraphId AC_edge_id;
+    const DirectedEdge* AC_edge = nullptr;
+    GraphId CA_edge_id;
+    const DirectedEdge* CA_edge = nullptr;
+    std::tie(AC_edge_id, AC_edge, CA_edge_id, CA_edge) =
+        findEdge(graph_reader, map.nodes, "AC", "C", baldr::GraphId{});
+    EXPECT_NE(AC_edge, nullptr);
+    EXPECT_NE(CA_edge, nullptr);
+
+    auto node_id = gurka::findNode(graph_reader, layout, "C");
+    const auto* node = graph_reader.nodeinfo(node_id);
+
+    uint32_t ntrans = node->local_edge_count();
+    for (uint32_t k = 0; k < ntrans; k++) {
+      if ((AC_edge->restrictions() & (1 << k))) {
+        EXPECT_EQ((AC_edge->restrictions() & (1 << k)), 16);
+      }
+    }
+  }
+}
+
+TEST(Standalone, FilterTestConsistencyTTHeadingsDriveability) {
+  constexpr double gridsize_metres = 50;
+
+  const std::string ascii_map = R"(
+            B
+            |
+            |
+            |
+     A------C------D
+           /|\
+          / | \
+         /  |  \
+        E   F   G
+    )";
+
+  const gurka::ways ways = {
+      {"AC", {{"highway", "secondary"}, {"osm_id", "100"}, {"name", "ABC Road"}}},
+      {"CB", {{"highway", "secondary"}, {"osm_id", "101"}}},
+      {"CD", {{"highway", "footway"}, {"osm_id", "102"}}},
+      {"CG", {{"highway", "footway"}, {"osm_id", "103"}, {"name", "ABC Road"}}},
+      {"CF", {{"highway", "secondary"}, {"osm_id", "104"}, {"name", "Test Road"}, {"oneway", "yes"}}},
+      {"CE", {{"highway", "secondary"}, {"osm_id", "105"}}},
+  };
+
+  auto layout = gurka::detail::map_to_coordinates(ascii_map, gridsize_metres, {-73.94913, 42.81490});
+  auto map =
+      gurka::buildtiles(layout, ways, {}, {}, work_dir,
+                        {{"mjolnir.admin", {VALHALLA_SOURCE_DIR "test/data/language_admin.sqlite"}},
+                         {"mjolnir.include_pedestrian", "false"}});
+  {
+    GraphReader graph_reader = GraphReader(map.config.get_child("mjolnir"));
+
+    GraphId AC_edge_id;
+    const DirectedEdge* AC_edge = nullptr;
+    GraphId CA_edge_id;
+    const DirectedEdge* CA_edge = nullptr;
+    std::tie(AC_edge_id, AC_edge, CA_edge_id, CA_edge) =
+        findEdge(graph_reader, map.nodes, "", "C", baldr::GraphId{}, 100);
+    EXPECT_NE(AC_edge, nullptr);
+    EXPECT_NE(CA_edge, nullptr);
+
+    GraphId CF_edge_id;
+    const DirectedEdge* CF_edge = nullptr;
+    GraphId FC_edge_id;
+    const DirectedEdge* FC_edge = nullptr;
+    std::tie(CF_edge_id, CF_edge, FC_edge_id, FC_edge) =
+        findEdge(graph_reader, map.nodes, "Test Road", "F", baldr::GraphId{});
+    EXPECT_NE(CF_edge, nullptr);
+    EXPECT_NE(FC_edge, nullptr);
+
+    const auto* node = graph_reader.nodeinfo(AC_edge->endnode());
+    /*auto tile1 = graph_reader.GetGraphTile(AC_edge->endnode());
```

Comment:
whats with this block comment? is it just for debugging and you want to remove it or?